src/frontend/src/lib/api/signer.api.ts

```diff
@@ -48,6 +70,24 @@ export const signPrehash = async ({
 	hash: string;
 	identity: OptionIdentity;
 }): Promise<string> => {
-	const { sign_prehash } = await getSignerActor({ identity });
-	return sign_prehash(hash);
+	const { signPrehash } = await signerCanister({ identity });
+
+	return signPrehash({ hash });
+};
+
+const signerCanister = async ({
+	identity
+}: {
+	identity: OptionIdentity;
+}): Promise<SignerCanister> => {
+	assertNonNullish(identity);
```

Comment:
any particular reson why you removed the i18n key?

Reply:
I didn't use the i18 store due to `no-svelte-store-in-api` (and hence future usage of the signer canister in a web worker). I can move `assertNonNullish(identity)` up to the places where signer API methods are called from, if we need to keep that message. WDYT?

Reply:
Ah right, good idea. Thx!

Reply:
Done.